python/awaazde/apis.py

```diff
@@ -82,10 +82,12 @@ def upload(self, file_path):
 
         return self._client.post(upload_url, **data)
 
-    def create_bulk(self, data):
+    def create_bulk(self, data,**kwargs):
         """
         This will create new object
         """
         bulk_url = self.url + "create_bulk/"
+        transform_using_template = kwargs.pop('transform_using_template')
```

Comment:
@ashwini-ad Based on this implementation, We are passing transform_using_template every time in request and the value of this will change based on passed data in kwargs. So I feel it's not a proper use of kwargs. Ideally, after the data parameter, You can pass another parameter called "transform_using_template" and specify the default value of False. So that it will be an explicit parameter of this method and everyone can see the same.